### ISSUE TITLE
Error Pug\Symfony\CssExtension

### ISSUE THREAD

**Issue by helveden** [pug-php/pug-symfony#62]
Hello,

I encountered an issue when:

`php bin/console c:c`

Result :
In ExtensionSet.php line 120:
Unable to register extension "Pug\Symfony\CssExtension" as extensions have already been initialized.

Thanks!

**Comment by kylekatarnls** [comment]
Hello 👋 

Can you provide the complete stack trace of the error running `php bin/console -v c:c`

THanks.

**Comment by helveden** [comment]
```
Exception trace:
  at C:\Sites\test-pug\vendor\twig\twig\src\ExtensionSet.php:120
 Twig\ExtensionSet->addExtension() at C:\Sites\test-pug\vendor\twig\twig\src\Environment.php:582
 Twig\Environment->addExtension() at C:\Sites\test-pug\vendor\pug-php\pug-symfony\src\Pug\Symfony\Traits\HelpersHandler.php:355
 Pug\PugSymfonyEngine->copyTwigFunctions() at C:\Sites\test-pug\vendor\pug-php\pug-symfony\src\Pug\Symfony\Traits\HelpersHandler.php:81
 Pug\PugSymfonyEngine->getRenderer() at C:\Sites\test-pug\vendor\pug-php\pug-symfony\src\Pug\Twig\Environment.php:62
 Pug\Twig\Environment->getRenderer() at C:\Sites\test-pug\vendor\pug-php\pug-symfony\src\Pug\Twig\Environment.php:142
 Pug\Twig\Environment->compileSource() at C:\Sites\test-pug\vendor\twig\twig\src\Environment.php:348
 Twig\Environment->loadTemplate() at C:\Sites\test-pug\vendor\pug-php\pug-symfony\src\Pug\Twig\Environment.php:205
 Pug\Twig\Environment->loadTemplate() at C:\Sites\test-pug\vendor\twig\twig\src\Environment.php:309
 Twig\Environment->load() at C:\Sites\test-pug\vendor\symfony\twig-bundle\CacheWarmer\TemplateCacheWarmer.php:49
 Symfony\Bundle\TwigBundle\CacheWarmer\TemplateCacheWarmer->warmUp() at C:\Sites\test-pug\vendor\symfony\http-kernel\CacheWarmer\CacheWarmerAggregate.php:95
 Symfony\Component\HttpKernel\CacheWarmer\CacheWarmerAggregate->warmUp() at C:\Sites\test-pug\vendor\symfony\framework-bundle\Command\CacheClearCommand.php:196
 Symfony\Bundle\FrameworkBundle\Command\CacheClearCommand->warmup() at C:\Sites\test-pug\vendor\symfony\framework-bundle\Command\CacheClearCommand.php:129
 Symfony\Bundle\FrameworkBundle\Command\CacheClearCommand->execute() at C:\Sites\test-pug\vendor\symfony\console\Command\Command.php:255
 Symfony\Component\Console\Command\Command->run() at C:\Sites\test-pug\vendor\symfony\console\Application.php:929
 Symfony\Component\Console\Application->doRunCommand() at C:\Sites\test-pug\vendor\symfony\framework-bundle\Console\Application.php:96
 Symfony\Bundle\FrameworkBundle\Console\Application->doRunCommand() at C:\Sites\test-pug\vendor\symfony\console\Application.php:264
 Symfony\Component\Console\Application->doRun() at C:\Sites\test-pug\vendor\symfony\framework-bundle\Console\Application.php:82
 Symfony\Bundle\FrameworkBundle\Console\Application->doRun() at C:\Sites\test-pug\vendor\symfony\console\Application.php:140
 Symfony\Component\Console\Application->run() at C:\Sites\test-pug\bin\console:42
```

**Comment by kylekatarnls** [comment]
Hello, can you please try to `composer require pug-php/pug-symfony:dev-fix/issue-62-double-register` to see if it fixes your issue?

**Comment by helveden** [comment]
No i have a same issues, sorry :)

**Comment by tombevers** [comment]
@kylekatarnls I was in the progress of upgrading a Symfony 4 application to Symfony 5.4 and I did encounter this bug. As you suggested I tried the fix. But this didn't solve the issue.

 php bin/console cache:clear -v
 
 `In ExtensionSet.php line 120:

  [LogicException]
  Unable to register extension "Pug\Symfony\CssExtension" as extensions have already been initialized.


Exception trace:
  at /application/vendor/twig/twig/src/ExtensionSet.php:120
 Twig\ExtensionSet->addExtension() at /application/vendor/twig/twig/src/Environment.php:600
 Twig\Environment->addExtension() at /application/vendor/pug-php/pug-symfony/src/Pug/Symfony/Traits/HelpersHandler.php:357
 Pug\PugSymfonyEngine->copyTwigFunctions() at /application/vendor/pug-php/pug-symfony/src/Pug/Symfony/Traits/HelpersHandler.php:81
 Pug\PugSymfonyEngine->getRenderer() at /application/vendor/pug-php/pug-symfony/src/Pug/Twig/Environment.php:62
 Pug\Twig\Environment->getRenderer() at /application/vendor/pug-php/pug-symfony/src/Pug/Twig/Environment.php:142
 Pug\Twig\Environment->compileSource() at /application/vendor/twig/twig/src/Environment.php:348
 Twig\Environment->loadTemplate() at /application/vendor/pug-php/pug-symfony/src/Pug/Twig/Environment.php:205
 Pug\Twig\Environment->loadTemplate() at /application/vendor/twig/twig/src/Environment.php:309
 Twig\Environment->load() at /application/vendor/symfony/twig-bundle/CacheWarmer/TemplateCacheWarmer.php:53
 Symfony\Bundle\TwigBundle\CacheWarmer\TemplateCacheWarmer->warmUp() at /application/vendor/symfony/http-kernel/CacheWarmer/CacheWarmerAggregate.php:99
 Symfony\Component\HttpKernel\CacheWarmer\CacheWarmerAggregate->warmUp() at /application/vendor/symfony/framework-bundle/Command/CacheClearCommand.php:244
 Symfony\Bundle\FrameworkBundle\Command\CacheClearCommand->warmup() at /application/vendor/symfony/framework-bundle/Command/CacheClearCommand.php:156
 Symfony\Bundle\FrameworkBundle\Command\CacheClearCommand->execute() at /application/vendor/symfony/console/Command/Command.php:298
 Symfony\Component\Console\Command\Command->run() at /application/vendor/symfony/console/Application.php:1058
 Symfony\Component\Console\Application->doRunCommand() at /application/vendor/symfony/framework-bundle/Console/Application.php:96
 Symfony\Bundle\FrameworkBundle\Console\Application->doRunCommand() at /application/vendor/symfony/console/Application.php:301
 Symfony\Component\Console\Application->doRun() at /application/vendor/symfony/framework-bundle/Console/Application.php:82
 Symfony\Bundle\FrameworkBundle\Console\Application->doRun() at /application/vendor/symfony/console/Application.php:171
 Symfony\Component\Console\Application->run() at /application/vendor/symfony/runtime/Runner/Symfony/ConsoleApplicationRunner.php:54
 Symfony\Component\Runtime\Runner\Symfony\ConsoleApplicationRunner->run() at /application/vendor/autoload_runtime.php:35
 require_once() at /application/bin/console:13

cache:clear [--no-warmup] [--no-optional-warmers]`

**Comment by tombevers** [comment]
This issue seems to occur when you have the following defined in services.yaml.

`Pug\PugSymfonyEngine:`
`        arguments:`
`            - '@kernel'`

We had the `Pug\PugSymfonyEngine` injected in a service. But `Twig\Environment` should be enough to render correctly.

**Comment by kylekatarnls** [comment]
Fixed in [4.0](https://github.com/pug-php/pug-symfony/releases/tag/4.0.0)
